packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-email/java/summaryofsteps.md

```diff
@@ -0,0 +1,9 @@
+## Summary of steps
+
+The following sequence of steps show the sign in with password and email flow.
```

Comment:
The following graphic shows the steps for the sign in with password and email flow: